Gauge/BarGauge: Added support for value mapping of "no data"-state to text/value (#20842)
grafana/grafana: docs/sources/features/panels/singlestat.md

{{< docs-imagebox img="/img/docs/v45/singlestat-value-options.png" class="docs-image--no-shadow" max-width="900px" >}}

> 1. **Stats**: The Stats field let you set the function (min, max, average, current, total, first, delta, range) that your entire query is reduced into a single value with. This reduces the entire query into a single summary value that is displayed.

Reviewer: If you'd like to improve further, I'd recommend using lazy list ordering, especially for longer lists, i.e.:

```
1. First item
1. Second item
1. Third item
```

<div class="clearfix"></div>

> > **_Pro-tip:_** Reduce the opacity on fill colors for nice looking panels.

Reviewer: ```suggestion
> **Pro-tip:** Reduce the opacity on fill colors for nice looking panels.
```

I don't think the emphasis is needed here.

Author: ouch, mybad 🙈 I have prettier formatting enabled so it will reformat the `md` code according to the prettier settings of the project.

What is our thoughts regarding this?

Value/Range to text mapping allows you to translate the value of the summary stat into explicit text. The text will respect all styling, thresholds and customization defined for the value. This can be useful to translate the number of the main Singlestat value into a context-specific human-readable word or message.

> If you want to replace the default "No data" text being displayed when no data is available. You can do this by adding a `value to text mapping` from `null` to your preferred custom text value.

Reviewer: ```suggestion
If you want to replace the default "No data" text being displayed when no data is available, add a `value to text mapping` from `null` to your preferred custom text value.
```

The first sentence is incomplete here.

Author: awesome feedback, thanks! Will update 🙏
